README.md

```diff
@@ -23,3 +23,8 @@ The `comm` subproject contains code for network related operations for RChain.
 ### Storage
 
 The `storage` subproject contains code related to the key-value storage of the RChain blockchain.
+
+### Filing Issues
+
+You can file issues in our Public Jira Instance here: https://rchain.atlassian.net/secure/CreateIssue!default.jspa
```

Comment:
Any way to get this to default to CORE/Bug instead of OPS/Story?

Reply:
Or better yet: have a Release situation where issues can be filed, complete which "Which release you on about?" selector, etc.? Might be more than JIRA wants to do for us.